templates/nextjs/package.json

```diff
@@ -15,7 +15,7 @@
     "@fuels/connectors": "^0.8.1",
     "@fuels/react": "^0.20.0",
     "fuels": "workspace:*",
-    "@tanstack/react-query": "^5.51.11",
+    "@tanstack/react-query": "^5.51.15",
```

Comment:
Should this be treated as a patch update to the template?

Reply:
Resolving to merge